index.html

```diff
@@ -0,0 +1,142 @@
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon2.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon3.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon4.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon5.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+    </div>
+    <button class="main-program-button">Join Save Cauvery 2022</button>
+    <h3 class="mi-hidden">SEE THE WHOLE PROGRAM</h3>
+  </section>
+
+  <section class="speakers" id="join">
+    <h3 class="speaker-h31">Featured Speakers</h3>
+    <hr class="speaker-hr">
+    <div class="speaker-container">
+    </div>
+    <button class="speaker-button">MORE <img class="button-image" src="Images/button-img.png"></button>
+</section>
+
+  <section class="partners" id="sponsor">
+    <h3>Partner</h3>
+    <hr id="body-hr3">
+    <div class="container text-center partners-mobile">
+      <div class="row" id="row1">
+        <div class="col">
+          <img src="Images/partner1.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner3.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner4.png">
+        </div>
+      </div>
+      <div class="row" id="row2">
+        <div class="col">
+          <img src="Images/partner5.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner2.png">
+        </div>
+      </div>
+    </div>
+    <div class="container text-center partners-desktop">
+      <div class="row" id="row12">
+        <div class="col">
+          <img src="Images/partner1.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner3.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner4.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner5.png">
+        </div>
+        <div class="col">
+          <img src="Images/partner2.png">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <footer class="footer">
+    <img id="footer-logo" src="Images/footer-logo1.png">
+    <div>
+      <h6>2022 Save Cauvery Tamilnadu</h6>
+      <h6>Some Rights Reserved</h6>
```

Comment:
- [ ] In the mobile design of the home page there isn't a footer section. Kindly fix this 🙏